graspologic/inference/latent_distribution_test.py

```diff
@@ -39,6 +51,10 @@ class LatentDistributionTest(BaseInference):
 
     Parameters
     ----------
+    A1, A2 : variable (see description of 'input_graph')
+        The two graphs, or their embeddings to run a hypothesis test on.
+        Expected variable type and shape depends on input_graph attribute
```

Comment:
you're pointing people to input_graph which talks about.fit and .predict, which no longer exist?..

Reply:
Addressed in lines 151-161